.github/workflows/deploy.yml

```
@@ -0,0 +1,41 @@
+name: Production Tag Deployment
+env:
+    VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+    VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+on:
+    push:
+        tags:
+            - "@bigcommerce/catalyst-core@latest"
+            - "@bigcommerce/catalyst-makeswift@latest"
+            - "@bigcommerce/catalyst-b2b-makeswift@latest"
+jobs:
+    Deploy-Tag:
+        runs-on: ubuntu-latest
+        steps:
```

We probably should add `concurrency` since it can be triggered by multiple tags at once.
```suggestion
        concurrency: 
          group: ${{ github.workflow }}-${{ github.ref_name }}
        steps:
```

---

Updated!